isic_kredit/ColorConstants.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ColorConstants.swift
+//  
+//
+//  Created by Petr Dusek on 9/02/2019.
+//
+
+import Foundation
+import UIKit
+
+struct ColorConstants {
+    static let BackgroundColor = UIColor(red:0.27, green:0.60, blue:0.72, alpha:1.0)
+    static let LabelsColor =  UIColor(red:0.70, green:0.83, blue:0.94, alpha:1.0)
+    static let TextFieldColor =  UIColor(red:0.22, green:0.51, blue:0.63, alpha:1.0)
+    static let LoginBackgroundColor = UIColor(red:0.57, green:0.86, blue:1.00, alpha:1.0)
```

Comment:
Proměnný píšem s malým na začátku, neva že jsou static - ty taky.

Reply:
Ano, nevim proc sem to psal velkyma 😄